dea_waterbodies/make_time_series.py

```diff
@@ -166,28 +166,65 @@ def get_shapes(config_dict: dict,
+              default='ALL',
+              help='sets the time range for the waterbody '
+              'timeseries queries. --time-span will default to ALL if no '
+              'other option is specified. If you select APPEND, then only '
+              'times since the latest dates in the waterbody timeseries will '
+              'be run. If --time-span = CUSTOM, then --start and --start must '
+              'also be specified.')
+@click.option('--start', type=str, default=None,
+              help='Date string. E.g. 2019-01-01. '
+              'The start date for the waterbody timeseries query. If --start '
+              'is provided --end must also be provided.')
+@click.option('--end', type=str, default=None,
+              help='Date string. E.g. 2019-12-01. '
+              'The end date for the waterbody timeseries query. If --end is '
+              'provided --start must also be provided.')
+@click.option('--missing-only/--not-missing-only', default=False,
+              help='Specifies whether you want to only run '
+              'waterbody polygons that DO NOT already have a .csv file '
+              'in the --output directory. The default option is to run '
+              'every waterbody polygon in the --shapefile file, and overwrite '
+              'any existing csv files.')
 @click.option('--state', type=click.Choice(
                 ['ACT', 'NSW', 'NT', 'OT', 'QLD', 'SA', 'TAS', 'VIC', 'WA']),
-              default=None)
-@click.option('--no-mask-obs/--mask-obs', default=False)
-@click.option('--all/--some', default=False)
+              default=None,
+              help='Run the analysis for selected '
+              'Australian states only. This option requires that each polygon '
+              'have an attribute called "STATE" that specifies one of the '
+              'specified state options.')
+@click.option('--no-mask-obs/--mask-obs', default=False,
+              help='Option to include uncertainties in the '
+              'output timeseries. if you specify --no-mask-obs then you will '
+              'only filter out timesteps with 100% invalid pixels. You will '
+              'also record the number invalid pixels per timestep.')
+@click.option('--all/--some', default=False,
+              help='Option to run a subset of the polygons in '
+              'the --shapefile, or --all of them (default). If --some, you '
+              'must also provide a list of ids using the ids argument.')
 @click.option('-v', '--verbose', count=True)
 @click.version_option(version=dea_waterbodies.__version__)
-def main(ids, config, shapefile, start, end, size,
-         missing_only, skip, time_span, output, state,
-         no_mask_obs, all, verbose):
-    """Make the waterbodies time series."""
+def main(ids, config, shapefile, start, end, missing_only, 
+         time_span, output, state, no_mask_obs, all, verbose):
+    """
+    Make the waterbodies time series. \n
```

Comment:
These \ns are not needed in a docstring.

Reply:
But when I ran the waterbodies-ts CLI, it munged all the text into one line. This was the only way I could format it so that it worked from the command line. 
Is there a better way?

Reply:
Without /n
![image](https://user-images.githubusercontent.com/19562367/126587683-21386451-1707-4646-967b-bcd0673c42fc.png)
With /n
![image](https://user-images.githubusercontent.com/19562367/126587702-07e113ed-afeb-4025-afc9-641c269a3698.png)

Reply:
oooh. Fair enough!